components/kyma-operator/pkg/kymahelm/kyma-helm-client.go

```diff
@@ -71,6 +75,37 @@ func (hc *Client) ReleaseStatus(rname string) (string, error) {
 	return strings.Replace(statusStr, `\n`, "\n", -1), nil
 }
 
+//IsReleaseDeletable returns true for release that can be deleted
+func (hc *Client) IsReleaseDeletable(rname string) (bool, error) {
+	isDeletable := false
+	maxAttempts := 3
+
+	err := retry.Do(
+		func() error {
+			statusRes, err := hc.helm.ReleaseStatus(rname)
+			if err != nil {
+				if strings.Contains(err.Error(), errors.ErrReleaseNotFound(rname).Error()) {
+					isDeletable = false
+					return nil
+				}
+				return err
+			}
+			isDeletable = statusRes.Info.Status.Code != release.Status_DEPLOYED
```

Comment:
What about status "DELETING"? Is such a release deletable, or Helm will throw back an error if we try to delete it?

Reply:
helm will throw an error and we will have retry, which would work if release was deleted